### PR TITLE
Bump Local packages, bump this add-on's version

### PR DESCRIPTION
**Pull request by kienstra**
* Bumps `@getflywheel/local` and `@getflywheel/local-components` to the latest
* Bumps this add-on's version

Fixes [LOC-5980](https://wpengine.atlassian.net/browse/LOC-5980)

[LOC-5980]: https://wpengine.atlassian.net/browse/LOC-5980?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Testing looks good.

3 of the 4 Atlas blueprints worked.

The Shopify blueprint had this error after creating it:

>Server Error
>Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.
>Learn more: https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor
>This error happened while generating the page. Any console logs will be displayed in the terminal window.

Screenshot 2024-03-29 at 5 02 58 PM

…but the fix should be in [wpengine/atlas-shopify-blueprint](https://github.com/wpengine/atlas-shopify-blueprint), I think.

`@getflywheel/local-components` look good:

Screenshot 2024-03-29 at 4 51 09 PM

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v16.16.0
+v18.18.0
```
Comment (kienstra) [inline .nvmrc:1]: To be the [same](https://github.com/getflywheel/flywheel-local/blob/b8650337bf628f6efadb771907aaacadd0c498d2/.nvmrc) as Local Core.

File: package.json [hunk 2]
```diff
@@ -62,16 +62,16 @@
 		"ts-jest": "^26.3.0",
 		"ts-loader": "^8.0.4",
 		"typescript": "^4",
-		"webpack": "^4.44.2",
-		"webpack-cli": "^3.3.12"
+		"webpack": "^5.91.0",
+		"webpack-cli": "^5.1.4"
```
Comment (kienstra) [inline package.json:66]: Webpack `4` didn't work with Node `18`, so I bumped it

File: webpack.config.js [hunk 3]
```diff
@@ -1,20 +1,19 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const path = require('path');
+const path = require('path'); // eslint-disable-line @typescript-eslint/no-var-requires
+const { merge } = require('webpack-merge'); // eslint-disable-line @typescript-eslint/no-var-requires
```
Comment (kienstra) [inline webpack.config.js:2]: This config didn't work with Webpack `5 `, so I updated it

**Review by nickcernis — APPROVED** [review]
Looking great, thank you, @kienstra ! I saw the same issue with the [wpengine/atlas-shopify-blueprint](https://github.com/wpengine/atlas-shopify-blueprint), but once I remove the nested `<a>` from the `<Link>`s it builds fine, so agree the issue is there.

Screenshot 2024-04-03 at 14 53 37
Screenshot 2024-04-03 at 14 56 02
Screenshot 2024-04-03 at 15 02 19
Screenshot 2024-04-03 at 15 04 16
Screenshot 2024-04-03 at 15 14 09

**Pull request merged**